ui/tests/helpers/general-selectors.ts

```diff
@@ -14,6 +14,7 @@ export const GENERAL = {
   headerContainer: 'header.page-header',
   icon: (name: string) => `[data-test-icon="${name}"]`,
   tab: (name: string) => `[data-test-tab="${name}"]`,
+  hdsTab: (name: string) => `[data-test-tab="${name}"] button`, // hds tabs are li elements and QUnit needs a clickable element so add button to the selector
```

Comment:
I was trying to find where I'd added this! I knew i had put it somewhere 😂 
https://github.com/hashicorp/vault/blob/4617af328b09fdcfb83a173bc130b994c542300b/ui/tests/helpers/tools-selectors.ts#L9

Reply:
Good find. Because this selector is needed outside of tools, I'll go ahead and replace those instances using tools-selectors and have them use the one off of general-selectors.

Reply:
thank you!